### PR TITLE
Fix: Controls do not show on iOS

### PR DESCRIPTION
**Pull request by DispatchCommit**
This attempts to remove code that prevents any controls from appearing on iOS. This may not fully resolve the issue though, as there may be an issue with both native and custom controls appearing on iOS devices.
However, it's a lot easier to remove / disable when they appear, and very difficult to understand what is preventing the appearance when they do not show at all.

this pr does the following:
- remove `video-js-parent--ios` class which hid control bar via css.
- remove `nativeControlsForTouch` option in favor of allowing for default behavior.
- remove `IS_IOS` const as the video.js library already defines a value that could be used instead, and there are no longer any references to it.
- Add `videojs.browser.IS_IOS` back to videojs-mobile-ui plugin as we should aim to have a unified experience across all devices. (also it might fix the controls not appearing issue)
- remove styling for `.video-js-parent--ios` class as we no longer use this class
- move `videoJsOptions.muted = startMuted;` to where `videoJsOptions` is initially defined.

## PR Checklist

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

- [x] Bugfix
- [ ] Feature
- [x] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: #5263 (maybe)

## What is the current behavior?

no controls on iOS

## What is the new behavior?

hopefully more controls on iOS. Possible too many controls.

## Other information

This may introduce a new bug. But that bug would be less troublesome than current behavior.
It is unclear why certain modifications were made to some settings and code by other devs, so this first pass attempts to simplify the problem first. 

It's possible that both native controls and custom video.js controls appear on iOS devices after this PR. 
If that occurs, debugging because much simpler in which dev tools can be used to trigger breakpoints for determining where in code the additional controls come from. 

This is easier to debug than trying to pinpoint which portion of these changes prevented or removed *all* the controls.

tldr: Testing on iOS hardware just needs to be done with these modifications to see if controls re-appear first, and I do not own an iPhone.

### CODE REVIEW

**Comment by seanyesmunt** [comment]
@DispatchCommit Have you looked at this again?

**Comment by seanyesmunt** [comment]
I'm still seeing this
image

**Review by seanyesmunt — CHANGES_REQUESTED** [review]
see screenshot
